test/InputOutput/finitedifference.jl

```diff
@@ -0,0 +1,49 @@
+using Test
```

Comment:
We need to include this to be run in the test suite somewhere. Also, can we can follow the convention and call this filename `unit_finitedifference.jl`?

Reply:
All the other files in the folder also didn't have the `unit_` prefix, so I changed the names of all the files in `test/InputOutput`

(And added to `runtests`)

Reply:
Ok, thanks!